imports/ui/case-wizard/case-wizard.jsx

```diff
@@ -55,6 +55,19 @@ class CaseWizard extends Component {
     }
   }
 
+  receivedUnitTitle = () => {
```

Comment:
There's no need for to make changes on the case-wizard. I'll introduce some changes to master in a PR soon which will allow you to just use "/case/new?unit=<unitId>" to populate this field